### PR TITLE
Fixed headers and modify script that checks files

### PR DESCRIPTION
**Pull request by jcalderin**

### CODE REVIEW

File: scripts/check_files_compliance.py [hunk 1]
```diff
@@ -113,26 +112,34 @@ def ignore(root, file):
...
+            or file == 'README.md' or file == 'LICENSE' or file == 'ContributionPolicy.txt' \
+            or file == 'CHANGES_NEXT_RELEASE' or file == 'compileInfo.h' or file == 'unittests_that_fail_sporadically.txt' \
+            or file == 'Vagrantfile' or file == "contextBroker.ubuntu":
         return True
     if 'scripts' in root and (file == 'cpplint.py' or file == 'pdi-pep8.py' or file == 'uncrustify.cfg' \
-        or file == 'cmake2junit.xsl'):
+                                      or file == 'cmake2junit.xsl'):
         return True
     if 'acceptance' in root and (file.endswith('.txt') or file.endswith('.json')):
         return True
```
Comment (fgalan) [inline scripts/check_files_compliance.py]: Up to here, all the changes in check_files_compliance.py seems to be indentation... Is that correct? Why intentation has changed?

File: scripts/check_files_compliance.py [hunk 2]
```diff
@@ -113,26 +112,34 @@ def ignore(root, file):
...
 
+
 def supported_extension(root, file):
     # FIXME: there should be a smarter way of doing this instead of so long or statement :)
-    if file.endswith('.py') or file.endswith('.cpp') or file.endswith('.h') or file.endswith('.xml')\
-        or file.endswith('.json') or file.endswith('.test') or file.endswith('.vtest') or file.endswith('.txt')\
-        or file.endswith('.sh') or file == 'makefile' or file == 'Makefile' or file.endswith('.spec') \
-        or file.endswith('.cfg') or file.endswith('.DISABLED') or file.endswith('.xtest') or file.endswith('.centos') \
-        or file.endswith('.js') or file.endswith('.jmx') or file.endswith('.vtestx'):
+    extensions = ['py', 'cpp', 'h', 'xml', 'json', 'test', 'vtest', 'txt', 'sh', 'spec', 'cfg', 'DISABLED', 'xtest',
```
Comment (fgalan) [inline scripts/check_files_compliance.py]: Much better this way :)

The commented block of code can be safely removed.

File: scripts/check_files_compliance.py (hunk 1 above)
Comment (jcalderin) [inline scripts/check_files_compliance.py]: Its the automatic PEP8 indentation done by pycharm... but in the cases of this "or"s it seems to be a bit rare... im going to refactor the or

File: scripts/check_files_compliance.py (hunk 2 above)
Comment (jcalderin) [inline scripts/check_files_compliance.py]: Deleted the comment block in  60d3475

File: scripts/check_files_compliance.py (hunk 1 above)
Comment (jcalderin) [inline scripts/check_files_compliance.py]: Fixed in 60d3475

**Comment by fgalan** [comment]
LGTM

**Pull request merged**